modules/api/src/main/java/org/apache/ignite/configuration/schemas/table/TablesConfigurationSchema.java

```diff
@@ -29,6 +29,10 @@
 @SuppressWarnings("PMD.UnusedPrivateField")
 @ConfigurationRoot(rootName = "table", type = ConfigurationType.DISTRIBUTED)
 public class TablesConfigurationSchema {
+    /** Global integer id counter. */
+    @Value(hasDefault = true)
+    public int globalIdCounter = 0;
```

Comment:
Is this the global table identifier? why is it not clearly named? what happens if it overflows?

Reply:
I'l expand the comment